lib/carin_for_blue_button_test_kit/custom_groups/visual_inspection_and_attestation/v200_server/11_crosscutting_group/coverage_reference_resource.rb

```diff
@@ -0,0 +1,31 @@
+module CarinForBlueButtonTestKit
+  class CoverageReferenceResourceTest < Inferno::Test
+    title 'Returns Coverage Reference Resource with correct data'
+
+    description <<~DESCRIPTION
+      The Health IT Module returns the Coverage Reference Resource with data that was effective as of the date of service of the claim.#{' '}
+      For example, the data will reflect the employer name in effect at that time.
+    DESCRIPTION
+
+    id :coverage_reference_resource
+
+    verifies_requirements 'hl7.fhir.us.carin-bb_2.0.0@87',
```

Comment:
91 is listed in out of scope reqs

Reply:
modified to be in scope. let Karl know. Will regenerate files.